AdyenComponents/PayTo/PayToPayIdentifier.swift

```diff
@@ -4,24 +4,26 @@
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
 /// Identifier options for PayTo, known as PayID.
-internal enum PayToPayIdentifier: String, CustomStringConvertible, CaseIterable {
+internal enum PayToPayIdentifier: String, CaseIterable {
     case phone
     case email
     case abn
     case organizationId
 
-    // TODO: Add translation
-    public var description: String {
+    public var localizedDescription: LocalizationKey {
```

Comment:
Since it's not a description anymore, `localizedKey` would make more sense, what do you think?

Reply:
Yes good point. I will check it.